src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -352,10 +352,10 @@ suspend fun handleMessage(
                 sm2013BehandlingsUtfallToipic,
                 loggingMeta)
 
-        val findNAVKontorService = FindNAVKontorService(receivedSykmelding, personV3, loggingMeta)
+        val findNAVKontorService = FindNAVKontorService(receivedSykmelding, personV3, norg2Client, arbeidsfordelingV1, loggingMeta)
 
-        val navKontorManuellOppgave = findNAVKontorService.findNavkontorTaskNr(arbeidsfordelingV1)
-        val navKontorLokalKontor = findNAVKontorService.findLocaleNavkontorNr(norg2Client)
+        val navKontorManuellOppgave = findNAVKontorService.finnBehandlendeEnhet()
```

Comment:
Skal disse to da hete behandlendeEnhet && lokaltNavkontor?

Reply:
Ifix